operators/README.md

```diff
@@ -142,20 +144,6 @@ tutor dev do importdemocourse
 tutor dev run cms ./manage.py cms update_course_outline course-v1:OpenedX+DemoX+DemoCourse
 ```
 
-### Extra credit: running the Frontend Plugin Framework example
-
-```bash
-git clone git@github.com:openedx/frontend-plugin-framework.git
-make requirements
-
-# On one screen
-npm run start
-
-# On another screen
-npm run start:example
-```
-
-
```

Comment:
Getting the FPF examples to work with tutor is still a bit tricky, and iirc requires CORS workarounds similar to the ones described in https://github.com/openedx/frontend-platform/issues/811. I figure just leaving this out and having people use examples in MFEs that work with `npm run dev` makes more sense.